app/web/features/search/FilterDialog.stories.tsx

```diff
@@ -16,10 +16,17 @@ const Template: Story<any> = () => {
   return (
     <>
       <FilterDialog
-        updateMapBoundingBox={() => jest.fn()}
+        setLocationResult={() => jest.fn()}
+        lastActiveFilter={0}
+        setLastActiveFilter={() => jest.fn()}
+        hostingStatusFilter={0}
+        setHostingStatusFilter={() => jest.fn()}
+        completeProfileFilter={true}
+        setCompleteProfileFilter={() => jest.fn()}
+        numberOfGuestFilter={undefined}
+        setNumberOfGuestFilter={() => jest.fn()}
```

Comment:
I'm curious about the jest functions outside of a test file. Do you us `component.stories.tsx` to represent a test?

Reply:
oh, didn't realize about this... I though was a test will change it to avoid confusion!

Reply:
done ✅